File: IT/Interfacce grafiche/C++/graphicHelloWorld/graphicHelloWorld.cpp
/* file graphicHelloWorld.cpp */

#include <QApplication>
#include <QLabel>

int main(int argc, char* argv[])
{
	/* crea l'applicazione principale */

	QApplication application(argc, argv);

	/* crea un'etichetta di testo */

	QLabel label("Ciao, mondo!");

	/* rende visibile l'etichetta */

	label.show();

	/* inizia il ciclo di vita dell'applicazione */

	return application.exec();
}

File: IT/2/Interfacce grafiche/Python/graphicHelloWorld.py
# file graphicHelloWorld.py

# supporto per Python 2 e Python 3
try:
	from tkinter import Frame, Label
except ImportError:
	from Tkinter import Frame, Label

# crea la finestra principale

frame = Frame()

# crea un'etichetta di testo

label = Label(frame, text="Ciao, mondo!")

# visualizza l'etichetta

label.pack()

# visualizza la finestra principale

frame.pack()

# inizia il ciclo di vita dell'applicazione

frame.mainloop()

